app/src/main/java/com/github/geohunt/app/ui/components/leaderboard/LeaderboardChips.kt

```diff
@@ -0,0 +1,52 @@
+package com.github.geohunt.app.ui.components.leaderboard
+
+/*
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.github.geohunt.app.ui.theme.geoHuntRed
+
+enum class LeaderboardChoice(val text: String) {
+    GLOBAL("Global"),
+    FRIENDS("Friends")
+}
+
+/**
+ * Draws the leaderboard chips used to select between global and friends
+ * leaderboards.
+ *
+ * This file can be ignored until different leaderboards are implemented.
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun LeaderboardChips() {
+    var selectedLeaderboard by remember {
+        mutableStateOf(LeaderboardChoice.GLOBAL)
+    }
+
+    @Composable
+    fun LeaderboardChip(choice: LeaderboardChoice, isSelected: Boolean) {
+        FilterChip(
+            onClick = { selectedLeaderboard = choice },
+            selected = isSelected,
+            colors = ChipDefaults.filterChipColors(
+                selectedBackgroundColor = geoHuntRed,
+                selectedContentColor = Color.White,
+                selectedLeadingIconColor = Color.White
+            )
+        ) {
+            Text(choice.text)
+        }
+    }
+
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(10. dp)
+    ) {
+        LeaderboardChip(LeaderboardChoice.GLOBAL, isSelected = selectedLeaderboard == LeaderboardChoice.GLOBAL)
+        LeaderboardChip(LeaderboardChoice.FRIENDS, isSelected = selectedLeaderboard == LeaderboardChoice.FRIENDS)
+    }
+}
+*/
```

Comment:
This is not debug code, but code that could be added once/if different leaderboards are implemented, such as "Global", "Friends", "Country", etc. This file is left for future usages.

The code was commented to remove it from code coverage, tests should be added if this file is uncommented. I'll add a comment at the top of the file to make it clearer.

Reply:
I feel like their would be a more "clean" solution to this issue, but I guess this is an acceptable solution (and don't see how we could do otherwise)